### DISCUSSION TITLE
xrdp service does not start on boot automatically

### DISCUSSION THREAD

**Discussion by amardeepl11** [neutrinolabs/xrdp#2877] [Q&A] [answered]
Hi All,

When i am rebooting my ubuntu 22, xrdp service does not start on boot. I had to manually restart the service.

xrdp service does not start on boot 

Please check and help me to fix this.

Thanks

**Comment by matt335672** [comment]
What does `systemctl status xrdp` say after a boot?

**Reply by amardeepl11** [reply, marked as answer]
Hi @matt335672  thanks reply, I have fixed the issue by doing this - https://www.suse.com/support/kb/doc/?id=000019917
Thanks
